solutions/gatekeeper-policies/guardrails/09-network-security-services/tests/cloudsql-test.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+  apiVersion: sql.cnrm.cloud.google.com/v1beta1
+  kind: SQLInstance
+  metadata:
+    name: sqlinstance-sample-private-ip
+    annotations:
+      config.kubernetes.io/local-config: 'true'      
+  spec:
+    databaseVersion: MYSQL_5_7
+    region: us-central1
+    settings:
+      tier: db-f1-micro
+      ipConfiguration:
+        ipv4Enabled: false
```

Comment:
Just a Q: why not enable ipv4?

Reply:
Hi Michael, the setting is to enable a public IPV4 address, which we are not allowing by default. This prevents deploying the sql instance with a public IP.

https://cloud.google.com/config-connector/docs/reference/resource-docs/sql/sqlinstance

Reply:
Thanks @lucstjean-ssc. I see it in the examples now. 👍 https://cloud.google.com/config-connector/docs/reference/resource-docs/sql/sqlinstance#private_ip_instance